bots/pr/src/test/java/org/openjdk/skara/bots/pr/CheckTests.java

```diff
@@ -41,6 +41,8 @@
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
 class CheckTests {
+    private static final String JEP_NUMBER = "customfield_10701";
```

Comment:
It should be possible to import this from JiraProject as well.

Reply:
Fixed.